src/proc/formats-converter.cpp

```diff
@@ -57,6 +57,40 @@ void formats_converter::drop_non_basic_formats()
     }
 }
 
+std::string to_string( std::shared_ptr< stream_profile_interface > profile )
+{
+    std::string res;
+
+    if( profile )
+    {
+        res += std::string( "(" ) + rs2_stream_to_string( profile->get_stream_type() ) + std::string( ")" );
```

Comment:
Wouldn't it be cleaner to just use `ostringstream` instead of all the `std::string` conversions? Or even an `operator<<`?

Reply:
Updated to `operator<<`